### DISCUSSION TITLE
2016 Audi A6, old firmware, will M.I.B alone allow me to patch enable Carplay or do I need to upgrade to latest firmware first?

### DISCUSSION THREAD

**Discussion by ronstone** [Mr-MIBonk/M.I.B._More-Incredible-Bash#458] [General]
I wasn't sure about this, but if my car has and older firmware (MHI2_US_AUG11_P0533) without Carplay enabled, can I install M.I.B. with this old firmware and apply the Carplay patch only with MIB and that's it?  Or must I first upgrade to latest firmware (MHI2_US_AU57x_K3341_1) first?   Wasn't sure if M.I.B. is all I need to enable Carplay, old firmware or not.   Sorry for the newb question.

**Comment by OSHKOSHKING** [comment]
Hi Ron, same question here. Did you get it figured out?

**Comment by jdomian** [comment]
Do either one of you have the MHI2_US_AUG11_P0533 file? I cannot find it anywhere anymore.

**Comment by ronstone** [comment]
I skipped the whole MIB thing and just went into red mode and updated to
the latest US AIO firmware, Bose fine, carplay enabled and didn't spend a
dime.
